test/integration/features/cmd-docker-env.feature

```diff
@@ -0,0 +1,41 @@
+@cmd-docker-env @command
+Feature: Command docker-env
+Command docker-env helps the user to set environmental variables which are needed force
+the usage of Minishift with docker command on VM host.
```

Comment:
This explanation of this feature file is not clear to me. Suggestion:  command `docker-env` sets docker environment variables for supported shells.`

Reply:
@LalatenduMohanty Thanks, that sentence is confused, missed that, your suggestion seems fine to me.